src/mutmap/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+find_package(Rscript)
+
+set(MUTMAP_DIR ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBEXECDIR}/mutmap)
+
+configure_file("${CMAKE_CURRENT_SOURCE_DIR}/mutmap.sh.in"
+  "${CMAKE_CURRENT_BINARY_DIR}/mutmap.sh" @ONLY)
+
+configure_file("${CMAKE_CURRENT_SOURCE_DIR}/dist/build.R"
+  "${CMAKE_CURRENT_BINARY_DIR}/build.R" @ONLY)
+
+configure_file("${CMAKE_CURRENT_SOURCE_DIR}/dist/bundle.js"
+  "${CMAKE_CURRENT_BINARY_DIR}/bundle.js" @ONLY)
+
```

Comment:
Can you explain your workflow to me? I'm not sure how node is only required by the developer and not during compiling/building.

Since you don't have permission to look at that repo, you can try looking at the source of the WhiteZed app: https://cartwrig.ht/apps/whitezed/.